


class customError(Exception):
    pass
class FileDoesNotExist(Exception):
    pass